src/components/va-alert/va-alert.tsx

```diff
@@ -48,6 +48,12 @@ export class VaAlert {
    */
   @Prop() closeable: boolean = false;
 
+  /**
+   * If true, the alert will be full width.
+   * Should be for emergency communication only.
+   */
+  @Prop() fullwidth: boolean = false;
```

Comment:
:thinking: If we label this as `fullWidth`, does the use then become `<va-alert full-width>`? (Hyphenated instead of two words smushed together?)

Reply:
Yeah, I'm pretty sure. I'll switch to that.

Reply:
https://github.com/department-of-veterans-affairs/component-library/pull/181/commits/fd1ba62f67af64455b2faa3d76c57fa8506a5506

![image](https://user-images.githubusercontent.com/2008881/134558618-f0c571ef-75a6-4f5e-b4fd-6e062d23c48a.png)